### PR TITLE
updated all queries and transactions

### PR DESCRIPTION
**Pull request by tanveerisonline**
### In this PR, I have done the followings:

- Modify schema.sql
- Add data to the species column
- Write queries

**Screenshots:**

### Inside transaction:-

1. Deleted all animals born after Jan 1st, 2022.
2. Created a savepoint for the transaction.
3. Updated all animals' weight to be their weight multiplied by -1.
4. Rollbacked to the savepoint.
5. Updated all animals' weights that were negative to be their weight multiplied by -1.
6. Committed transaction.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Query to update and read the animals table data */
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```
Comment (bobb-Rob) [inline queries.sql]: Very well done in making this project to this point. 

- [ ] You should have a query to verify that a change was made - `SELECT species from animals;` before and after ROLLBACK. Kindly add this.

File: queries.sql [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name Like '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
```
Comment (bobb-Rob) [inline queries.sql]: - [ ] Based on the requirements, you are expected to have a query that will verify that a change was made. That query should be before and after the COMMIT as shown below 👇🏾 .

```sql
SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

File: queries.sql [hunk 3]
```diff
@@ -1,8 +1,54 @@
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```
Comment (bobb-Rob) [inline queries.sql]: - [ ] The same thing is expected here. There should be a query before and after ROLLBACK like below 👇🏾 

```sql
BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS; -- verify that the delete occured

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS; -- verify that the rollback happened
```

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @tanveerisonline, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (danmainah) [reply queries.sql]: - [ ] @tanveerisonline   you haven't implemented the changes requested to have a query to verify changes made before and after rollback when updating the animals' species to unspecified. Kindly add the query and also add screenshots for the same.

File: queries.sql (hunk 2 above)
Comment (danmainah) [reply queries.sql]: - [ ] @tanveerisonline   you haven't implemented the changes requested to have a query to verify changes made before and after the commit. Kindly add the query and also add screenshots for the same.

File: queries.sql (hunk 3 above)
Comment (danmainah) [reply queries.sql]: - [ ]  @tanveerisonline   you haven't implemented the changes requested to have a query to verify changes made before and after the rollback. Kindly add the query and also add screenshots for the same.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @tanveerisonline , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow was correctly used
✔️  Good PR and Description


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @tanveerisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good implementations ✔️ 
- Implement `CRs` suggestions ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**